flexmeasures_openweathermap/cli/schemas/weather_sensor.py

```diff
@@ -35,11 +43,22 @@ def validate_name_is_supported(self, name: str):
 
     @validates_schema(skip_on_field_errors=False)
     def validate_name_is_unique_in_weather_station(self, data, **kwargs):
-        if "name" not in data or "latitude" not in data or "longitude" not in data:
+        if (
+            "name" not in data
+            or "latitude" not in data
+            or "longitude" not in data
+            or "asset_id" not in data
```

Comment:
This seems wrong now.
As we're now allowing wither a location or an asset ID, this if-condition will always be true and this validation never happens.

Anyway, I feel that the logic in this validation function is not in the right place. We can just as well place lines 62 through 69 in the CLI command.

Reply:
Then we drop this whole validation since these have the core logic.

Reply:
I have kept the function, but moved those to cli part.